py3status/modules/wwan.py

```diff
@@ -324,8 +325,9 @@ def post_config_hook(self):
 
         self.bus = SystemBus()
         self.init = {'ip': [], 'sms_message': []}
-        self.last_notification = None
+        self.last_notification = self.py3.storage_get('last_notification')
```

Comment:
Destroy `py3status.data` (it's usually in same place as your `py3status.conf` and try again. See what happens when this starts up without any storage... cuz we may want something like this to deal with it.
```
self.last_notification = self.py3.storage_get('last_notification') or None
```
Be careful by killing py3status first, destroy storage, then start. See if it's causing problem.

Reply:
Nevermind about this. I think this is fine because maybe we want `None`. For `{}`, `[]`, or `''`, we would do `example = storage_get('last_example') or {}` instead.